.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,35 +1,40 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: ''
-labels: ''
-assignees: ''
-
+title: ""
+labels: ""
+assignees: ""
 ---
 
 **Describe the bug**
+
 - A clear and concise description of what the bug is.
 
 **To Reproduce**
+
 Steps to reproduce the behavior:
+
 1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+1. Click on '....'
+1. Scroll down to '....'
```

Comment:
Why are all of these 1s?

Reply:
it's because of the following:

```
MD029 Ordered list item prefix
```

It's recommended to use 1s for the ordered lists. The advantage is that when you add a new item somewhere between the items, you wouldn't need to re-order them

Reply:
Oh interesting. I just saw the rendered document. Looks like the list is automatically numbered correctly. Nice!